### DISCUSSION TITLE
Github Pages - How can I disable the page temporarily

### DISCUSSION THREAD

**Discussion by kkhandelwal** [community/community#33271] [Other Feature Feedback, Questions, & Ideas] [answered]
Github Pages - How can I disable the page temporarily or how to check if my page is disabled

thanks,

**Comment by melvincwng** [comment, marked as answer]
@kkhandelwal, go to your code repository, then click 'Settings' > 'Pages' > Find the '...' button & click 'Unpublish site'.

![GitHub Pages unpublish site image](https://docs.github.com/assets/cb-50834/images/help/pages/unpublish-site.png)

For more information, please refer to the [Official GitHub Documentation link](https://docs.github.com/en/pages/getting-started-with-github-pages/unpublishing-a-github-pages-site) over here.

Hope this clarifies and answers your question 😄

**Reply by queenofcorgis** [reply]
@melvincwng thanks for sharing this and the screenshot - @kkhandelwal did this answer your question, if so don't forget to mark as answer ✅ !
